spec/index.html

```diff
@@ -466,8 +466,8 @@ <h3>Quoted Literals</h3>
         <a href="#cp-apostrophe"><code title="apostrophe">'</code></a>,
         <a href="#cp-triple-quote"><code>"""</code></a>, or
         <a href="#cp-triple-apostrophe"><code>'''</code></a>);
-        a sequence of permitted characters, <a href="#numeric">numeric escapes</a>,
-        and/or <a href="#string">string escapes</a>; and a final delimiter (matching the initial delimiter).
+        a sequence of permitted characters, a <a>numeric escape sequences</a>,
```

Comment:
```suggestion
        a sequence of permitted characters, a <a>numeric escape sequence</a>,
```

Reply:
Fix made, making the all plural.  "sequence of As, Bs, and/or Cs"